Products/Poi/events.py

```diff
@@ -86,7 +85,11 @@ def remember_tracker(new_issue, event):
 
 
 def assign_id(new_issue, event):
-    """Auto-increment ID numbers"""
+    """Auto-increment ID numbers when they are created.
+       Don't run on copied issues
+    """
+    if new_issue.id.find('copy') >= 0:
+        return
```

Comment:
Should we (can we) just disable “Copy” for the `Issue` type? It wouldn’t make sense outside the context of its tracker, so where else are you going to put it? And it sounds like putting it back in the same place just messes up the ID.